dhcpv4/option_ip_address_lease_time.go

```diff
@@ -1,11 +1,15 @@
 package dhcpv4
 
 import (
+	"math"
 	"time"
 
 	"github.com/u-root/u-root/pkg/uio"
 )
 
+// MaxLeaseTime is the maximum lease time that can be encoded.
+var MaxLeaseTime = math.MaxUint32 * time.Second
```

Comment:
what is this for?

Reply:
Convenience for when I want to give a long lease